plugins/Installation/FormFirstWebsiteSetup.php

```diff
@@ -86,7 +86,7 @@ function init()
  * Timezone validation rule
  *
  */
-class Rule_isValidTimezone extends HTML_QuickForm2_Rule
+class RuleIsValidTimezone extends HTML_QuickForm2_Rule
```

Comment:
Why not move this to a separate file?

Reply:
I wanted to avoid making too many changes. And imho we should at some point completely refactor the installation forms to get rid of the QuickForm2 library.